doc-content/enterprise-only/installation/nfs-server-configure_proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='nfs-server-configure_{context}']
```

Comment:
Minor suggestion - some of the IDs are without the doc type designator - con/proc/ref. Should we add these?

Reply:
@csherrar I'm just afraid of breaking something by changing the ID of existing modules.